recipes/flickrhistory/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "flickrhistory" %}
+{% set version = "0.3.0.dev1" %}
```

Comment:
This one falls under https://github.com/conda-forge/cfep/blob/main/cfep-05.md

Reply:
We have since uploaded a stable version to PyPi, I’ve update the recipe